br/pkg/lightning/backend/external/byte_reader.go

```diff
@@ -29,7 +29,7 @@ import (
 var (
 	// ConcurrentReaderBufferSizePerConc is the buffer size for concurrent reader per
 	// concurrency.
-	ConcurrentReaderBufferSizePerConc = int(4 * size.MB)
+	ConcurrentReaderBufferSizePerConc = int(2 * size.MB)
```

Comment:
Will it affect the performance?

Reply:
In current way of using membuf.Pool, `ConcurrentReaderBufferSizePerConc `must be changed otherwise the ingest step will OOM. And 20T test shows the performance has no drop. I will test about more cases and use two pools of different sizes in future PR.